pgjdbc/src/test/java/org/postgresql/test/jdbc2/DatabaseMetaDataTest.java

```diff
@@ -433,7 +530,7 @@ void multiColumnForeignKeys(BinaryMode binaryMode) throws Exception {
 
   @MethodSource("data")
   @ParameterizedTest(name = "binary = {0}")
-  void sameTableForeignKeys(BinaryMode binaryMode) throws Exception {
+  void sameTableForeignKeys_whenSchemaArgEmpty_expectNoResults(BinaryMode binaryMode) throws Exception {
     Connection con1 = TestUtil.openDB();
```

Comment:
As we update code, can you change this to use try with resources and get rid of the closeDB below ?